src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3695,18 +3695,25 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     ? super List
+     *     List&lt;? super List&gt; list;
      * </pre>
      *
      * <p>parses as:</p>
      *
      * <pre>
-     * +--TYPE_ARGUMENT
-     *     |
-     *     +--WILDCARD_TYPE (?)
-     *     +--TYPE_LOWER_BOUNDS
-     *         |
-     *         +--IDENT (List)
+     *--VARIABLE_DEF -> VARIABLE_DEF
```

Comment:
Hi @haanhvu, javadoc supports html tags, so that `->` can be considered as closing tag, 
thats why we use characters like [here](https://github.com/checkstyle/checkstyle/pull/9378/files)

Reply:
actually you already have used them in your example :)

Reply:
oh yeah. I didn't check the AST carefully enough. I'll update now.

Reply:
@timurt I made the change you requested.

However, the CI tests are still failing. I found out the cause is probably some steps in the development process are not done properly.

Should I amend this PR, or close this then restart the development process then create a new PR?